### PR TITLE
Add installation instruction to the README

### LINKED ISSUE #4: Provide wiki on what how to use this toolkit
#### Describe the documentation issue:
- [ ] The code - the attribute of a method or class
- [x] The wiki

#### What is missing/incorrect?

Given the presence of another programming language, the wiki should provide details on what is the content of this repository.

### LINKED ISSUE #32: Document the procedure to correctly install the toolkit
#### Describe the documentation issue:
- [ ] The code - the attribute of a method or class
- [x] The wiki

### PR DESCRIPTION
**Pull request by epignatelli**
### Issues addressed by this PR

Closes #32
Closes #4 

Added the installation procedure to the README of the repo.

### CODE REVIEW

**Comment by epignatelli** [comment]
Hey @BingWangUS ! 

Whenever you have time, would you mind looking at this?
In case you are too busy, do let me know, I'll ask someone else!

File: README.md [hunk 1]
```diff
@@ -1,12 +1,37 @@
...
+To install the Python_Toolkit:
+1. Compile:  
+    1. [BHoM](https://github.com/BHoM/BHoM)  
+    1. [BHoM_Engine](https://github.com/BHoM/BHoM_Engine)  
+    1. [BHoM_Adapter](https://github.com/BHoM/BHoM_Adapter)   
+    1. [BHoM_UI](https://github.com/BHoM/BHoM_UI)  
+	  1. [Rhinoceros_Toolkit](https://github.com/BHoM/Rhinoceros_Toolkit)  
+	  1. [Grasshopper_Toolkit](https://github.com/BHoM/Grasshopper_Toolkit)  
+	  1. Python_Toolkit (this repo)
+1. Open ai UI of your choice (e.g. Grasshopper)
```
Comment (BingWangUS) [inline README.md:12]: ai -> a

File: README.md [hunk 2]
```diff
@@ -1,12 +1,37 @@
...
+1. Run the `InstallPythonToolkit` component and wait for the installation to finish.
+1. he installation has succeeded if the packages installed are:
```
Comment (BingWangUS) [inline README.md:14]: he -> The
"if the packages installed are:" -> "if the installed packages include:"

File: README.md [hunk 3]
```diff
@@ -1,12 +1,37 @@
...
+	  - Python 3.7
+  	- jupyter
+  	- matplotlib
+  	- Python_Toolkit
+1. Restart your UI
```
Comment (BingWangUS) [inline README.md:19]: Restart your UI (in this case, both Rhino and Grasshopper)

File: README.md [hunk 4]
```diff
@@ -1,12 +1,37 @@
...
+1. Compile:  
+    1. [BHoM](https://github.com/BHoM/BHoM)  
+    1. [BHoM_Engine](https://github.com/BHoM/BHoM_Engine)  
+    1. [BHoM_Adapter](https://github.com/BHoM/BHoM_Adapter)   
+    1. [BHoM_UI](https://github.com/BHoM/BHoM_UI)  
+	  1. [Rhinoceros_Toolkit](https://github.com/BHoM/Rhinoceros_Toolkit)  
+	  1. [Grasshopper_Toolkit](https://github.com/BHoM/Grasshopper_Toolkit)  
+	  1. Python_Toolkit (this repo)
+1. Open ai UI of your choice (e.g. Grasshopper)
+1. Run the `InstallPythonToolkit` component and wait for the installation to finish.
```
Comment (BingWangUS) [inline README.md:13]: `BH.Engine.Python.Compute.InstallPythonToolkit`

File: README.md [hunk 5]
```diff
@@ -1,12 +1,37 @@
...
+1. he installation has succeeded if the packages installed are:
+	  - Python 3.7
+  	- jupyter
+  	- matplotlib
+  	- Python_Toolkit
+1. Restart your UI
+
+To check that all went ok you can run the component `BH.Engine.Python.Compute.Import` with input a string `Python_Toolkit`,
```
Comment (BingWangUS) [inline README.md:21]: To check whether all went well you can run the component `BH.Engine.Python.Compute.Import` with a input of a string as `Python_Toolkit`,

File: README.md (hunk 1 above)
Comment (epignatelli) [reply README.md:12]:
```suggestion
1. Open a UI of your choice (e.g. Grasshopper)
```

File: README.md (hunk 2 above)
Comment (epignatelli) [reply README.md:14]:
```suggestion
1. The installation has succeeded if the install packages include:
```

File: README.md (hunk 3 above)
Comment (epignatelli) [reply README.md:19]:
```suggestion
1. Restart your UI - if using Grasshopper, restart Rhinoceros and Grasshopper
```

File: README.md (hunk 4 above)
Comment (epignatelli) [reply README.md:13]:
```suggestion
1. Run the `BH.Engine.Python.Compute.InstallPythonToolkit` component and wait for the installation to finish.
```

File: README.md (hunk 5 above)
Comment (epignatelli) [reply README.md:21]:
```suggestion
To check whether all went well you can run the component `BH.Engine.Python.Compute.Import` with a input of a string as `Python_Toolkit`,
```

**Comment by epignatelli** [comment]
Thanks very much @BingWangUS !

**Pull request merged**